src/resources/database/createCloudFiles.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE cloudFiles (
+    id TEXT NOT NULL PRIMARY KEY,
+    parentId TEXT NOT NULL,
```

Comment:
This field can be NULL if file/folder within the root folder.

Reply:
Using of empty string instead of NULL is preferable, it simplifies work with queries